picketlink/src/test/java/org/wildfly/extension/picketlink/subsystem/RoleGeneratorDeclarationUnitTestCase.java

```diff
@@ -22,10 +22,15 @@
 
 package org.wildfly.extension.picketlink.subsystem;
 
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.containsString;
 import static org.junit.Assert.assertFalse;
```

Comment:
Unused import

Reply:
oops, I am sorry I forgot to push the latest version